pkg/cluster/prepare.go

```diff
@@ -20,6 +20,16 @@ const (
 	prepareManifestFile = "prepare.yaml"
 )
 
+// FluxCRDs are CRDs which prepare is checking if they are present in the cluster or not.
+var FluxCRDs = []string{
+	"buckets.source.toolkit.fluxcd.io",
+	"gitrepositories.source.toolkit.fluxcd.io",
+	"helmcharts.source.toolkit.fluxcd.io",
+	"helmreleases.helm.toolkit.fluxcd.io",
+	"helmrepositories.source.toolkit.fluxcd.io",
+	"kustomizations.kustomize.toolkit.fluxcd.io",
```

Comment:
is this discover-able by referencing there respective packages? E.g. https://github.com/weaveworks/profiles/blob/9f749b8ceb974e4b51471d29412e8437b90300c3/pkg/profile/helm.go#L49-L50

Reply:
Oh cool! Maybe. I'll check it out!

Reply:
Done :D